app/models/github_fetcher/null_issue_response.rb

```diff
@@ -0,0 +1,15 @@
+module GithubFetcher
+  class NullIssueResponse
```

Comment:
Good call. I was thinking of moving in this direction, then apparently forgot :)